src/games/marooned.rs

```diff
@@ -467,7 +469,7 @@ impl GameState {
     /// let removed: Vec<Position> = game.removed_positions().collect();
     /// assert_eq!(removed, vec![pos]);
     /// ```
-    pub fn removed_positions(&self) -> impl Iterator<Item = Position> + Clone + '_ {
+    pub fn removed_positions(&self) -> impl Iterator<Item = Position> + '_ {
```

Comment:
There is an open issue in `im` where their iterators don't implement clone, but the new version hasn't been pushed
https://github.com/bodil/im-rs/issues/134